src/models/person.py

```diff
@@ -0,0 +1,20 @@
+import uuid
+from typing import List, Optional
+
+from .base import Base
+from .film import Film
+
+
+class Person(Base):
+    """
+    Represents a person associated with a film.
+
+    Attributes:
+    - name (str): The name of the person.
+    - films (Optional[List[Film]]): List of films associated with the film (if available).
+
+    """
+
+    id: uuid.UUID
+    name: Optional[str]
```

Comment:
1. стоит переименовать name -> full_name
2. type full_name скорее всего строго str. 
```suggestion
    full_name: str
```

Reply:
+